dev-tools/packer/docker/xgo-image-deb6/beats-builder/Dockerfile

```diff
@@ -2,18 +2,23 @@ FROM tudorg/xgo-deb6-1.8.1
 
 MAINTAINER Tudor Golubenco <tudor@elastic.co>
 
-# Get libpcap binaries for linux
+# Get libpcap-32 binaries from a DEB file
 RUN \
 	mkdir -p /libpcap && \
     wget http://archive.debian.org/debian/pool/main/libp/libpcap/libpcap0.8-dev_1.1.1-2+squeeze1_i386.deb && \
```

Comment:
Yeah, we should, but we'd need to x-compile it from x86_64 to x86, which I wasn't sure how to do, and because the original bug doesn't seem to affect x86, I left it for later.